doc/src/bond_oxdna.rst

```diff
@@ -73,8 +98,6 @@ commands:
    *oxdna2/hbond* and an additional Debye-Hueckel pair style
    *oxdna2/dh* have to be defined. The same applies to the oxRNA2
    :ref:`(Sulc1) <Sulc01>` styles.
-   The coefficients in the above example have to be kept fixed and cannot
-   be changed without reparameterizing the entire model.
 
```

Comment:
You removed this comment, but kept it below. Shall we rephrase this in a way that once LJ or real units have been chosen, coefficients need to be kept fixed?